package.json

```diff
@@ -16,11 +16,14 @@
         "hoek": "3.x.x"
     },
     "devDependencies": {
-        "lab": "7.x.x"
+        "lab": "8.x.x",
+        "markdown-toc": "^0.12.3"
```

Comment:
No, markdown-toc. Semver doesn't really apply the same way prior to v1.x.x. Can you drop the ^